chingsley/ireporter
# Ft fetch postnewrecord #163057994

#### What does this PR do?
This pull request deals with implementation of fetch to POST/CREATE a new record.

#### Description of Task to be completed?
- create the report.js file to implement the fetch
- using fetch consume the API with the route /api/v1/red-flags and /api/v1/interventions simultaneously, calling using the 'POST' method, and setting up the FormData().
- Display error messages by calling the 'showDialogMsg()' function with the right arguments
- on successful fetch, display a success message.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- at the frontend, test the functionalities by trying to sign-up as a new user. Then using your email and password, head over to the login page and sign in. This action will take you to the 'reporthistory' page which handles the fetch to 'GET' all records. Click on 'report a case' in the navigation link, then fill the form and submit. 

#### What are the relevant pivotal tracker stories
#163057994
